client/utils/rest.go

```diff
@@ -1,12 +1,30 @@
 package utils
 
 import (
+	"fmt"
 	"net/http"
 )
 
+const (
+	queryArgDryRun = "dry_run"
+)
+
 // WriteErrorResponse prepares and writes a HTTP error
 // given a status code and an error message.
 func WriteErrorResponse(w *http.ResponseWriter, status int, msg string) {
 	(*w).WriteHeader(status)
 	(*w).Write([]byte(msg))
 }
+
+// WriteGasEstimateResponse prepares and writes an HTTP
+// response for transactions simulations.
+func WriteSimulationResponse(w *http.ResponseWriter, gas int64) {
+	(*w).WriteHeader(http.StatusOK)
+	(*w).Write([]byte(fmt.Sprintf(`{"gas_estimate":%v}`, gas)))
```

Comment:
Can we add a space after the key? Or maybe even define an inline simple struct (might be overkill)?

Reply:
This seems sufficiently simple imo, I think an inlined struct is more confusing.

Reply:
I guess browsers would render it correctly anyway.